RELEASENOTES.md

```diff
@@ -77,6 +77,39 @@ usage. (Intel IPU E2100 target only).
   * Supports maximum of 2 vports and each port supports maximum of 4 rx and tx
    queues
 
+### K8s Recipe
```

Comment:
To keep it consistent with previous releases, please update to call this

`Kubernetes Infrastructure Offload Recipe`